src/routes/routes.tsx

```diff
@@ -0,0 +1,15 @@
+import { createBrowserRouter, createRoutesFromElements, Route } from 'react-router-dom';
+import App from '../App';
+import { HomePage, LoginPage, NotFoundPage } from '../pages';
+
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route path="/" element={<App />}>
```

Comment:
Create a file called `RootLayout` in the layouts folder then export it in the index file there
```suggestion
    <Route path="/" element={<RootLayout />}>
```